src/main/java/com/fismobile/appium/base/BaseClass.java

```diff
@@ -75,15 +73,13 @@ private WebDriver setup()
                 } catch (MalformedURLException e) {
                     e.printStackTrace();
```

Comment:
Never print the stacktrace. Either log and/or handle the exception (fallback management, set variables, etc, depending on the scenario).